src/Illuminate/Foundation/Testing/RefreshDatabase.php

```diff
@@ -109,7 +109,6 @@ public function beginDatabaseTransaction()
                 $connection->unsetEventDispatcher();
                 $connection->rollBack();
                 $connection->setEventDispatcher($dispatcher);
-                $connection->disconnect();
```

Comment:
Shouldn't it just been `$database->purge($name);`?

Reply:
Fixed in https://github.com/laravel/framework/pull/49327/commits/71d57c294d408fce140e091ca1c71b39f3bb95b8